src/wizard/wizard.scss

```diff
@@ -102,15 +106,15 @@
       }
     }
 
-    > .iui-wizards-step-current {
-      > .iui-wizards-step-title {
+    .iui-wizards-step.iui-wizards-step-current {
```

Comment:
Sure need to change this selector?

Reply:
I've reverted it back to try to keep the changes minimal.